src/components/StatedBalanceForm.tsx

```diff
@@ -14,6 +15,7 @@ class StatedBalanceForm extends React.Component<Props, State> {
   render() {
     const callback = (value: number) => {
       this.props.balanceService.addBalance(value);
+      RoutingService.instance.goToGraph();
```

Comment:
We need to think what to do about this instances (they should be injected), but it is out of scope of this PR.